client/analytics/report/products/index.js

```diff
@@ -23,6 +24,11 @@ export default class ProductsReport extends Component {
 	render() {
 		const { path, query } = this.props;
 
+		const itemsLabel =
+			'single_product' === query.filter && !! query.products
+				? __( '%s variations', 'wc-admin' )
```

Comment:
What is the convention on multi-line ternaries?

Reply:
Per https://make.wordpress.org/core/handbook/best-practices/coding-standards/javascript/#multi-line-statements I understand it's ok using them, right?